src/wdio/services/TerraService.js

```diff
@@ -58,5 +57,10 @@ export default class TerraService {
     };
     chai.Assertion.addMethod('accessible', chaiMethods.accessible);
     chai.Assertion.addMethod('matchReference', chaiMethods.matchReference);
+    // IE driver takes a longer to be ready for browser interactions
```

Comment:
just curious, have an idea how much longer on average?

Reply:
Searching online says give it 10 seconds. I don't have a set time. Sometimes this pause isn't needed because the browser is ready, sometimes its slower.